libs/ui/paginator/src/lib/paginator/paginator.component.ts

```diff
@@ -315,6 +317,18 @@ export class TsPaginatorComponent implements OnChanges, AfterViewInit {
   @Output()
   public readonly recordsPerPageChange = new EventEmitter<number>();
 
+  /**
+   * Emit an event when previous page button clicked
+   */
+  @Output()
+  public readonly prevPage = new EventEmitter();
+
+  /**
+   * Emit an event when next page button clicked
+   */
+  @Output()
+  public readonly nextPage = new EventEmitter<number>();
```

Comment:
Why does previous not emit anything, but next emits a `0`?

Reply:
whoops... it was purposely changed to debug something. forgot to remove...